temba/channels/models.py

```diff
@@ -472,6 +475,16 @@ def add_twilio_messaging_service_channel(cls, org, user, messaging_service_sid,
                               name=messaging_service_sid, address=None, config=config)
 
     @classmethod
+    def add_twiml_api_channel(cls, org, user, country, address, config):
+        role = SEND + RECEIVE + CALL + ANSWER
```

Comment:
@teehamaral TwiML API only supports `CALL + ANSWER` for now. Will this affect the tests?